examples/react-native/__tests__/intro.test.js

```diff
@@ -51,7 +51,10 @@ it('renders the ListView component', () => {
     .create(
       <ListView
         dataSource={dataSource}
-        renderRow={rowData => <Text>{rowData}</Text>}
+        renderRow={rowData =>
+          <Text>
+            {rowData}
+          </Text>}
```

Comment:
Yeah, we used to respect whatever the user did like objects but people complained that it wasn't consistent enough. We looked through a lot of jsx and it seemed like most common was to always expand it.

In general, test files are harder to print because they only exercise edge cases to the code, so the quality from prettier is generally poorer unfortunately :(